### PR TITLE
[INLONG-6935][Dashboard] Add Key-Value format for InlongStream

### LINKED ISSUE #6935: [Improve][Dashboard] Add Key-Value format for InlongStream
### Description

_No response_

### InLong Component

InLong Dashboard

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by kinfuy**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #6935

### CODE REVIEW

File: inlong-dashboard/src/metas/streams/common/StreamDefaultInfo.ts [hunk 1]
```diff
@@ -105,6 +105,10 @@ export class StreamDefaultInfo implements DataWithBackend, RenderRow, RenderList
           label: 'CSV',
           value: 'CSV',
         },
+        {
+          label: 'Key-Value',
+          value: 'Key-Value',
```
Comment (leezng) [inline inlong-dashboard/src/metas/streams/common/StreamDefaultInfo.ts:110]: Should the value communicated with the manager be `KV` here?
Such as: 
https://github.com/apache/inlong/blob/45d98689f146431258eefdd9148cde64daf9a020/inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/stream/InlongStreamRequest.java#L61

Comment (dockerzhang) [reply inlong-dashboard/src/metas/streams/common/StreamDefaultInfo.ts:110]: We need to confirm how to communicate with the manager.

Comment (healchow) [reply inlong-dashboard/src/metas/streams/common/StreamDefaultInfo.ts:110]: It is found that the Manager does not support it yet.
Suggest using `KV` when communicating with the Manager.

**Review by healchow — APPROVED** [review]

**Comment by ZuoFengZhang** [comment]
The value of `datatype` configured on the front end is not passed to StreamSource, the default is csv format:


https://github.com/apache/inlong/blob/master/inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/pulsar/PulsarSourceOperator.java#L123

https://github.com/apache/inlong/blob/master/inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/kafka/KafkaSourceOperator.java#L109

**Comment by dockerzhang** [comment]
> The value of `datatype` configured on the front end is not passed to StreamSource, the default is csv format:
> 
> https://github.com/apache/inlong/blob/master/inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/pulsar/PulsarSourceOperator.java#L123
> 
> https://github.com/apache/inlong/blob/master/inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/kafka/KafkaSourceOperator.java#L109

Could you add another PR to make sure the manager supports the KV type?

**Review by leezng — APPROVED** [review]

**Comment by ZuoFengZhang** [comment]
LGTM

**Comment by ZuoFengZhang** [comment]
> > The value of `datatype` configured on the front end is not passed to StreamSource, the default is csv format:
> > https://github.com/apache/inlong/blob/master/inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/pulsar/PulsarSourceOperator.java#L123
> > https://github.com/apache/inlong/blob/master/inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/kafka/KafkaSourceOperator.java#L109
> 
> Could you add another PR to make sure the manager supports the KV type?

I have another pr, just fixed it.

https://github.com/apache/inlong/pull/7001

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**